src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/FlywayConfiguration.java

```diff
@@ -25,10 +25,10 @@ public Flyway flyway()
     {
         Flyway flyway = new Flyway();
         flyway.setDataSource(dataSource);
-        flyway.setTable("${artifactId}_schema");
-        flyway.setSchemas("flyway_migrations");
+        flyway.setTable(flywaySettings.getTable());
+        flyway.setSchemas(flywaySettings.getSchemas());
```

Comment:
Ich würde es lieber nicht tun, da wir unter Umständen in Zukunft mehr als einen Server je DB-Typ haben.